pydis_site/templates/home/index.html

```diff
@@ -19,21 +19,21 @@ <h1 class="is-size-1">Who are we?</h1>
         <div class="column is-half">
           <p>
             We're a large community focused around the Python programming language.
-            We believe anyone can learn programming, and are very dedicated to helping
-            novice developers take their first steps into the world of code. We also
+            We believe anyone can learn to code, and are very dedicated to helping
+            novice developers take their first steps into the world of programming. We also
             attract a lot of expert developers who are seeking friendships, collaborators,
-            or looking to hone their craft by teaching and getting involved in the community.
+            or to hone their craft by teaching and getting involved with the community.
```

Comment:
I'm not a native speaker of English, but I prefer "getting involved in the community" over "getting involved with the community". [A comparative search for the two idioms](https://trends.google.com/trends/explore?q=involved%20in%20the%20community,involved%20with%20the%20community) shows that the former is far more popular, but that both are recognized by dictionaries. [This english.stackexchange thread](https://english.stackexchange.com/questions/267751/is-it-involved-with-or-involved-in/267787) also makes me think `in` is more appropriate than `with` here, as they seem to convey a slightly different meaning.

Reply:
I think actually that Tizzy's suggestion here was to use `within`.

Reply:
I think it should be `in`. Also [this](https://books.google.com/ngrams/graph?content=getting+involved+in+the+community%2Cgetting+involved+with+the+community&year_start=1800&year_end=2000&corpus=15&smoothing=3&share=&direct_url=t1%3B%2Cgetting%20involved%20in%20the%20community%3B%2Cc0%3B.t1%3B%2Cgetting%20involved%20with%20the%20community%3B%2Cc0#t1%3B%2Cgetting%20involved%20in%20the%20community%3B%2Cc0%3B.t1%3B%2Cgetting%20involved%20with%20the%20community%3B%2Cc0).